block/manager.go

```diff
@@ -72,6 +72,13 @@ type Manager struct {
 	// retrieveCond is used to notify sync goroutine (SyncLoop) that it needs to retrieve data
 	retrieveCond *sync.Cond
 
+	// whether or not the centralized sequencer is currently building a block
+	buildingBlock     bool
+	startedBuilding   uint32
```

Comment:
i don't think you need this, since it is not read anywhere.

Reply:
`buildingBlock` is used but I removed `startedBuilding`